src/app/settings/keyboard-mapping-settings/components/action-selector/action-selector.component.ts

```diff
@@ -11,6 +11,8 @@ import { Tv2Action } from '../../../../shared/models/tv2-action'
 export class ActionSelectorComponent implements OnInit {
   @Input() public selectedAction?: Tv2Action
   @Output() public actionSelected: EventEmitter<Tv2Action> = new EventEmitter()
+  @Input() public hasError?: boolean
+  @Input() public errorMessage?: string
```

Comment:
As mentioned in my other comment, the ActionSelector is not what should be marked as required, so these fields needs to be removed again.

Reply:
Fixed